common/remote/sockfile.cpp

```diff
@@ -1883,6 +1884,16 @@ class CRemoteFilteredFileIOBase : public CRemoteBase, implements IFileIO
          */
         return 0;
     }
+// IRemoteFileIO
+    virtual void addVirtualField(const char *fieldName, const char *fieldValue) override
```

Comment:
picky: really addVirtualInformation since you're not actually adding a virtual field.

Reply:
I've changed it addVirtualFieldMapping